src/scss/components/ads/_ad-control-bar.scss

```
@@ -3,13 +3,14 @@
 
 // Keep this mixin here for now since it is specific to the ad control bar
 @mixin hidden-animated-collapse-and-fade-out($duration: $animation-duration, $original-height: 2em) {
-  transition: opacity $duration ease-out, height $duration ease-out;
+  transition: opacity $duration ease-out, height $duration ease-out, margin-bottom $duration ease-out;
   opacity: 1;
   height: $original-height;
 
   &.#{$prefix}-hidden {
     opacity: 0;
     height: 0;
+    margin: 0;
```

You're using `margin-top` but here you're resetting all of the `margins`

```suggestion
    margin-top: 0;
```

---

Address in 82f484a546487ee6781c1e49d3e34ab715c8b8b9